React-Template-master/src/theme/overrides/Typography.tsx

```diff
@@ -1,95 +1,194 @@
-                    fontSize: '24px !important',
-                    lineHeight: '40px',
-                    color: '#000000'
+                    fontSize: '24px',
+                    lineHeight: '40px'
                 },
                 h6: {
-                    fontSize: '14px',
-                    lineHeight: '24px',
-                    letterSpacing: '0.3px',
-                    color: NEUTRAL.default
+                    fontWeight: 700,
+                    fontSize: '20px',
+                    lineHeight: '32px'
                 },
                 body1: {
-                    fontSize: '14px !important',
-                    lineHeight: '24px',
                     fontWeight: 400,
+                    fontSize: '16px',
+                    lineHeight: '28px',
                     letterSpacing: '0.3px'
                 },
 
                 body2: {
-                    height: '24px',
-                    fontWeight: 700,
+                    fontWeight: 400,
                     fontSize: '14px',
                     lineHeight: '24px',
                     letterSpacing: '0.3px'
                 },
                 button: {
                     fontWeight: 700,
-                    color: theme.palette.primary.light,
-                    fontSize: '14px',
-                    lineHeight: '20px',
-                    letterSpacing: '0.5px'
+                    fontSize: '16px',
+                    lineHeight: '24px',
+                    letterSpacing: '0.5px',
+                    textTransform: 'capitalize'
                 },
 
                 subtitle1: {
-                    width: '360px',
-                    height: '32px',
                     fontWeight: 700,
-                    fontSize: '22px !important',
+                    fontSize: '22px',
                     lineHeight: '32px',
-                    textAlign: 'center',
                     letterSpacing: '0.1px',
-                    color: '#28293D'
+                    textTransform: 'capitalize'
                 },
                 subtitle2: {
+                    fontWeight: 700,
+                    fontSize: '18px',
+                    lineHeight: '32px',
+                    letterSpacing: '0.1px',
+                    textTransform: 'capitalize'
+                },
+                caption: {
                     fontWeight: 400,
-                    fontSize: '14px !important',
-                    lineHeight: '24px',
-                    textAlign: 'center',
+                    fontSize: '12px',
+                    lineHeight: '16px',
                     letterSpacing: '0.3px'
+                },
+                overline: {
+                    fontWeight: 400,
+                    fontSize: '12px',
+                    lineHeight: '16px',
+                    letterSpacing: '1.5px',
+                    textTransform: 'uppercase'
                 }
             }
         }
     };
 };
+
+export const h7 = {
```

Comment:
Instead of export multiple typography put all under cutomTypography and export it.